packages/spore/src/dob/config/predefined.ts

```diff
@@ -0,0 +1,89 @@
+import { Decoder } from "../helper";
+
+const TESTNET_DECODERS = {
+  DOB0: {
+    code_hash: {
+      type: "code_hash",
+      hash: "0x13cac78ad8482202f18f9df4ea707611c35f994375fa03ae79121312dda9925c",
+    },
+    type_id: null,
+    type_script: null,
+  },
+  DOB1: {
```

Comment:
We can choose one for the devs. Experienced devs can handle these configs by themselves.